From 186ec82e033d8c718d760d14945535a9d7e160fd Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Sun, 29 Jan 2023 21:00:27 -0700
Subject: [PATCH] Fix cmake languages

When no language is set cmake defaults to C and CXX, since we support
C by itself we should set the project language to C so that we don't
get a compiler error when building without a CXX toolchain.

Fixes:
CMake Error at CMakeLists.txt:5 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://github.com/intel/ARM_NEON_2_x86_SSE/pull/62]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e4b8980..8f58e79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 # Modification: Copyright (C) 2017 David Hirvonen
 
 cmake_minimum_required(VERSION 3.0)
-project(NEON_2_SSE VERSION 1.0.0)
+project(NEON_2_SSE VERSION 1.0.0 LANGUAGES C)
 
 add_library(${PROJECT_NAME} INTERFACE)
 
-- 
2.34.1

